.gitignore

```diff
@@ -20,3 +20,18 @@ scripts/chroot_execvp
 .DS_Store
 *~
 venv
+
+#### Development Environment ####
+
+# VsCode IDE
+.vscode 
+# symbolic link exercises to /srv/courses
+exercises
+# automatic generated keys
+grader/ajax_key.py
+grader/secret_key.py
+
+# The word "deprecated" was added to the files and directories that use sandbox.
+# The word "deprecated" will help to identify the files that can be removed 
+# definetely.
+*deprecated*
```

Comment:
People normally do not have any files that contain "deprecated" in the file name, so I don't think that the deprecated rule should be in the `.gitignore` file. There is a way to ignore files on your computer without editing the `.gitignore` file that is shared with other developers too: `.git/info/exclude` works like `.gitignore`, but the exclude file is not shared with others (i.e., committed to the repository).

https://git-scm.com/docs/gitignore
https://stackoverflow.com/questions/22906851/when-would-you-use-git-info-exclude-instead-of-gitignore-to-exclude-files

Reply:
Done!